source/utils/merge.ts

```diff
@@ -1,32 +0,0 @@
-import {URL} from 'url';
```

Comment:
This is no longer needed because all the necessary options are merged by hand (I mean they are merged still automatically but these are explicit Got options - people are responsible for merging their own options).